### PR TITLE
As_json error in Event

### PR DESCRIPTION
**Pull request by davidwessman**
I missed out the options attribute in the ```as_json```-method, apparently my tests didn't check this either.

Ideas for new tests would be great, #102.

### CODE REVIEW

**Comment by davidwessman** [comment]
The event-calendar is currently empty and we should merge this and adress the testing in a new PR.

File: app/models/event.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def ical
     e
   end
 
-  def as_json
+  def as_json(options = {})
```
Comment (davidwessman) [inline app/models/event.rb]: Hmm, perhaps this is the issue? It should just accept attributes to be able to run it.

Comment (henrikssn) [inline app/models/event.rb]: Try `def as_json(_)`

Comment (henrikssn) [inline app/models/event.rb]: `def as_json(_ = {})` should work :)

Comment (davidwessman) [inline app/models/event.rb]: What about ```as_json(*)```?

**Comment by davidwessman** [comment]
Changed to ```as_json(*)``` from suggestion by Hound.

Ready to merge, again...

**Comment by henrikssn** [comment]
This PR was deployed to Production. Reference: <a href='http://samson.fsektionen.se/projects/web/deploys/47' target='_blank'>62cc3b4</a>

**Pull request merged**